### DISCUSSION TITLE
How to make text in item/row visible but not being searched in -dmenu?

### DISCUSSION THREAD

**Discussion by owzim** [davatorium/rofi#1949] [Q&A]
I want to show some additional info for each item, but don't want it to be searched. Is that posssible?

**Comment by DaveDavenport** [comment]
No this is not possible in dmenu mode.  Only the other way around is currently supported.

**Reply by owzim** [reply]
@DaveDavenport thanks for the reply. Is it a feasible feature or totally out of scope? Do you need me to discuss more, why this could be useful?

**Reply by DaveDavenport** [reply]
Its an feasible request, I don't think more information is needed.

**Reply by DaveDavenport** [reply]
There is a work-around-ish option I just realize with the 'display' option.

```bash
echo -en "testing\0display\x1ftesting (monkey)\n" | rofi -dmenu
```

is this sufficient for your use-case?
